test/include/test-phone.js

```diff
@@ -0,0 +1,162 @@
+const https = require('https')
```

Comment:
This kind of test is great but please, leave this file for another separate PR. Let's focus on the MediaConnection in this PR.